articles/quickstarts/index.md

```diff
@@ -19,17 +19,144 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
 - APIs for Python and .NET languages (C#, F#, and VB.NET) for running quantum programs written in Q#
 - Tools to facilitate your development
 
-Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, or through Jupyter Notebooks with the IQ# Jupyter kernel.
-They can also be paired with a host program written in a .NET language (typically C#) or Python, enabling you to call quantum operations from inside a classical program.
+Q# programs can run as standalone applications using Visual Studio Code or Visual Studio, through Jupyter Notebooks with the IQ# Jupyter kernel, or paired with a host program written in a .NET language (typically C#) or Python. You can also run Q# programs online using [Codespaces](https://online.visualstudio.com/) or [MyBinder.org](https://mybinder.org/). 
 
-The workflows for each of these setups are described and compared at [Ways to run a Q# program](xref:microsoft.quantum.guide.host-programs).
+## Installation options
+
+You can use the QDK three ways:
+
+- [Install the QDK locally](#install-the-qdk-locally)
+- [Use the QDK online](#use-the-qdk-online)
+- [Use a QDK Docker image](#use-the-qdk-with-docker)
+
+## Install the QDK locally
+
+You can develop Q# code in most of your favorites IDEs, as well as integrate Q# with other languages such as Python and .NET (C#, F#).
+
+|&nbsp; | **VS Code<br>(2019 or later)**| **Visual Studio<br>(2019 or later)** | **Jupyter Notebooks** | **Command line**|
+|:-----|:-----:|:-----:|:-----:|:-----:|
+|**OS** |Cross-platform |Windows only |Cross-platform |Cross-platform |
+|<br>**Q# standalone** |<br>&#10004; |<br> &#10004; | <br>&#10004; |<br> &#10004; |
+|**Q#  plus Python** |&#10004; |&#10006; |&#10004; |&#10004; |
```

Comment:
Why is "Q# plus Python" listed as available for VS Code but not for Visual Studio? I think our level of support is the same in both environments?

Reply:
That may be my fault; we've really not done the work to test and document things with PTVS, such that I had suggested Python as a VS Code–based experience.